service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -43,26 +47,60 @@ public class LandingZoneService {
   private final LandingZoneJobService azureLandingZoneJobService;
   private final LandingZoneManagerProvider landingZoneManagerProvider;
   private final LandingZoneDao landingZoneDao;
+  private final SamService samService;
 
   public LandingZoneService(
       LandingZoneJobService azureLandingZoneJobService,
       LandingZoneManagerProvider landingZoneManagerProvider,
-      LandingZoneDao landingZoneDao) {
+      LandingZoneDao landingZoneDao,
+      SamService samService) {
     this.azureLandingZoneJobService = azureLandingZoneJobService;
     this.landingZoneManagerProvider = landingZoneManagerProvider;
     this.landingZoneDao = landingZoneDao;
+    this.samService = samService;
   }
 
-  public AsyncJobResult<DeployedLandingZone> getAsyncJobResult(String jobId) {
+  /**
```

Comment:
In general I tried to add some Javadoc for the public methods of `LandingZoneService` (just to help myself out).